http-gateway/web/src/containers/app/AppRest.js

```diff
@@ -0,0 +1,6 @@
+import { fetchApi } from '@/common/services'
+
+export const getAppWellKnownConfiguration = wellKnowConfigUrl =>
+  fetchApi(`${wellKnowConfigUrl}/.well-known/configuration`, {
```

Comment:
consider adding this loading module in future to shared lib as it's used also by the d2d client app